ci/scripts/verible-format.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# A wrapper around Verible format, used for CI.
+
+set -e
```

Comment:
Remove this line, or you'll never get into the `if` check below.

Reply:
That works, but I'd probably suggest doing this instead:
```sh
set -e

util/verible-format.sh || {
  echo "Oh no!"
  ...
  exit 1
}
```

Reply:
Implemented rswarbrick's suggestion.